artifacts_test.py

```diff
@@ -325,6 +327,20 @@ def test_scylla_service(self):
             with self.subTest("check ENA support"):
                 assert self.node.ena_support, "ENA support is not enabled"
 
+        if backend in ["gce", "aws", "azure"] and self.params.get("use_preinstalled_scylla"):
+            with self.subTest("check Scylla IO Params"):
+                try:
+                    validator = IOTuneValidator(self.node)
+                    validator.validate()
+                    send_iotune_results_to_argus(
+                        self.test_config.argus_client(),
+                        validator.results,
+                        self.node,
+                        self.params
+                    )
+                except Exception:  # pylint: disable=broad-except # noqa: BLE001
+                    self.log.warning("IOTuneValidator failed", exc_info=True)
```

Comment:
let's raise Error event, otherwise we won't know about it

Reply:
Done